crawler.py

```diff
@@ -260,7 +270,7 @@ def run_all(driver, data_folder):
         report = json.load(rep)
 
```

Comment:
I would put the filtering here, as otherwise if it is in a function called `create_diff_report` it makes the filtering part of the diffing, but they are conceptually two different steps.